sapi/fpm/fpm/fpm_status.c

```diff
@@ -427,8 +427,26 @@ int fpm_status_handle_request(void) /* {{{ */
 				short_post = "";
 			} else {
 				full_separator = "";
-				full_pre = "";
-				full_syntax = "";
+				full_pre =
+					"# HELP phpfpm_process_start_since_seconds_total The number of seconds since the process started.\n"
+					"# TYPE phpfpm_process_start_since_seconds_total counter\n"
+					"# HELP phpfpm_process_requests_total The total number of requests served.\n"
+					"# TYPE phpfpm_process_requests_total counter\n"
+					"# HELP phpfpm_process_requests_seconds_total The total time in seconds spent serving requests.\n"
+					"# TYPE phpfpm_process_requests_seconds_total counter\n"
+					"# HELP phpfpm_process_last_request_cpu The percentage of cpu of the last request. This will be 0 if the process is not Idle because the calculation is done when the request processing is complete.\n"
+					"# TYPE phpfpm_process_last_request_cpu gauge\n"
+					"# HELP phpfpm_process_last_request_memory_bytes The maximum amount of memory consumed by the last request. This will be 0 if the process is not Idle because the calculation is done when the request processing is complete.\n"
+					"# TYPE phpfpm_process_last_request_memory_bytes gauge\n"
+					"# HELP phpfpm_process_last_request_content_length The maximum amount of memory consumed by the last request. This will be 0 if the process is not Idle because the calculation is done when the request processing is complete.\n"
+					"# TYPE phpfpm_process_last_request_content_length gauge\n";
+				full_syntax =
+					"phpfpm_process_start_since_seconds_total{pool=\"%16$s\",pid=\"%1$d\",state=\"%2$s\"} %4$lu\n"
```

Comment:
I don't think this is going to work - this is C so you cannot put positions here. Think the person implementing the initial OpenMetrics tried to use the same thing.